src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6526,6 +6526,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Module initializer cannot be attributed with 'UnmanagedCallersOnly'.</value>
     <comment>UnmanagedCallersOnly is not localizable.</comment>
   </data>
+  <data name="WRN_OnlyOneOfGetHashCodeAndEqualsIsDefined" xml:space="preserve">
```

Comment:
>WRN_OnlyOneOfGetHashCodeAndEqualsIsDefined [](start = 14, length = 42)

Warnings typically need a corresponding `_Title`. I expect a test is likely complaining about that.